src/karaoke.js

```diff
@@ -0,0 +1,101 @@
+/* globals CURRENT_ANIMATION_INFO */
+
+/**
+ * Append a word wrapped in a span to the provided parent node.
+ */
+function appendWord(parent, word, textVariant) {
+    const wordSpan = document.createElement('span')
+    wordSpan.textContent = word.text
+    wordSpan.className = `individual-word staged-word ${textVariant} no-margin text-height-1-1`
+    parent.appendChild(wordSpan)
+    return wordSpan
+}
+
+
+/**
+ * Find the quote that is currently being spoken from the utterances.
+ */
+function getCurrentQuote(utterances, time) {
```

Comment:
Nit: Might be worth sticking with the `utterance` naming

Reply:
Actually quote is being used in a few places below, given the time constrain, don't worry with updating the name